_includes/v22.1/cdc/avro-limitations.md

```diff
@@ -1,3 +1,7 @@
+- [Decimals](decimal.html) must have precision specified.
+- [`BYTE`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) (or its alias `BYTEA`) is often used to store machine-readable data. When you stream these types through a changefeed with [`format=avro`](create-changefeed.html#format), CockroachDB does not encode or change the data. However, Avro clients can often include escape sequences so the data is in a printable format. A potential solution, is to hex-encode `BYTEA` values when initially inserting them into CockroachDB. This will ensure that you can consistently decode the hexadecimal. Note that this process will increase record size. 
```

Comment:
The CockroachDB column type affected is BYTES, as well as its aliases BYTEA and BLOB. These all get encoded as the Avro type BYTES. Note that unlike other types in SQL or Avro, BYTES is pluralized--there's no singular BYTE type.

Otherwise, technical details LGTM. Suggest removing the comma after "a potential solution." Also, I feel like maybe it'd read better with some kind of description of why the escape sequences and printable format can be a problem requiring a solution, like "...in a printable format, which can interfere with deserialization."

Reply:
OK, I've edited the types + link here — do you think the small adjustment I have made reflects your point here? Yes, thanks for catching the typos on bytes and the errant comma there. Incorporated your other suggestions. I also added that mapping to the table as it wasn't there. Alphabetized while I was in the area.